lib/process_settings/abstract_monitor.rb

```diff
@@ -129,12 +127,17 @@ class << self
 
       def clear_instance
         @instance = nil
+        @default_instance = nil
```

Comment:
Note BTW that we're in a base class here, with `self` methods. Inheritance does not apply here. So we need to be careful.

Reply:
Maybe add a comment calling that out so that future developers are aware as well?